src/executor/load_executor.py

```diff
@@ -50,4 +50,6 @@ def exec(self):
                 data.append({'id': id, 'data': frame})
                 id += 1
         # Hook for the storage engine
-        append_rows(self.node.table_metainfo, data)
+        # We currently use create to empty exsiting table.
+        StorageEngine.create(self.node.table_metainfo)
+        StorageEngine.write_row(self.node.table_metainfo, data)
```

Comment:
Move this code chunk inside the video_reader loop. So basically we read a batch we store that batch. And can we change the API name to write_batch? That will remove confusion between row and batch.